iff --git a/spec/requests/user_pages_spec.rb b/spec/requests/user_pages_spec.rb
index 053e66b..68db7e9 100644
--- a/spec/requests/user_pages_spec.rb
+++ b/spec/requests/user_pages_spec.rb
@@ -41,6 +41,10 @@ describe "signup" do
         fill_in "Confirmation", with: "foobar"
       end
 
+      it "should create a user" do
+        expect { click_button submit }.to change(User, :count).by(1)
+      end
+
       describe "after saving the user" do
         before { click_button submit }
         let(:user) { User.find_by_email('user@example.com') }
@@ -48,18 +52,12 @@ describe "signup" do
         it { should have_selector('title', text: user.name) }
         it { should have_selector('div.alert.alert-success', text: 'Welcome') }
         it { should have_link('Sign out') }
-      end
 
-      it "should create a user" do
-        expect { click_button submit }.to change(User, :count).by(1)
-      end
-
-      describe "followed by signout" do
-        before { click_link "Sign out" }
-        it { should have_link('Sign in') }
+        describe "followed by signout" do
+          before { click_link "Sign out" }
+          it { should have_link('Sign in') }
+        end
       end
-
-
     end
   end